apollo-gradle-plugin/src/main/kotlin/com/apollographql/apollo/gradle/internal/ApolloDownloadSchemaTask.kt

```diff
@@ -66,7 +67,11 @@ abstract class ApolloDownloadSchemaTask : DefaultTask() {
         .url(urlBuilder.build())
         .build()
 
-    val response = OkHttpClient().newCall(requestBuilder.build()).execute()
+    val response = OkHttpClient.Builder()
+        .connectTimeout(System.getProperty("okHttp.connectTimeout","10").toLong(), TimeUnit.SECONDS)
+        .readTimeout(System.getProperty("okHttp.readTimeout","10").toLong(), TimeUnit.SECONDS)
```

Comment:
Thanks for the contribution 👍 . What about making the default a bit bigger like "60" ? Most of the times, this will be called from the dev machine or from CI so I think it's more important that the task actually downloads something compared to speed.

Reply:
I made the change through Github UI 😄